mindsdb/integrations/handlers/cohere_handler/README.md

```diff
@@ -0,0 +1,76 @@
+# Cohere Handler
+
+Cohere ML handler for MindsDB provides interfaces to connect with Cohere ML ia APIs and pull Cohere ML Capabilites into MindsDB.
+
+## Cohere
+
+Cohere platform provides solutions to many natural language use cases, including classification, semantic search, paraphrasing, summarization, and content generation. 
+In this handler,python client of cohere api is used and more information about this python client can be found (here)[https://docs.cohere.com/reference/about]
+
+
+## Implemented Features
+
+- [x] Cohere ML Handler
+  - [x] Support Generate
+  - [x] Support Detect_language 
+  - [x] Support Summarize
+
+## Example Usage
+
+The first step is to create a ML Engine with the new `cohere` engine.
+
+~~~~sql
+CREATE ML_ENGINE Cohere_ML_Engine
+FROM cohere
+USING
+  api_key = 'your_api_key';
+~~~~
+
+### Detect_language 
+
+~~~~sql
+CREATE MODEL mindsdb.cohere_language_detector
+PREDICT language
+USING
+  task = 'language-detection',
+  column = 'text',
+  engine = 'Cohere_ML_Engine',
+  api_key = 'your_api_key'
+
+SELECT text, language
+FROM mindsdb.cohere_language_detector
+WHERE text = 'Здравствуй, Мир';
+~~~~
+
+
+### Summarize
+
+~~~~sql
+CREATE MODEL mindsdb.cohere_text_summarization
+PREDICT summary
+USING
+  task = 'text-summarization',
+  column = 'text',
+  engine = 'Cohere_ML_Engine',
```

Comment:
Shouldn't this be 'cohere' ?

Reply:
Agreed, let's change the name of the engine to `cohere`

Reply:
Resolved the comment; thanks for reviewing